utils/index.js

```diff
@@ -21,3 +23,17 @@ const generateCustomerCreatedMessage = msg => ({
     firstName: msg.customer.firstName,
   },
 });
+
+const generateOrderCreatedMessage = msg => ({
```

Comment:
I think the code would be a bit cleaner if you extract `order` and `fromEmail` on the parameters signature like this:
```
({ order, fromEmail }) => ({
  to: order.customerEmail,
  from: fromEmail,
})
```